src/get-authorization-url.ts

```diff
@@ -4,9 +4,9 @@ import { GetAuthURLOptions } from './interfaces.js';
 import { headers } from 'next/headers';
 
 async function getAuthorizationUrl(options: GetAuthURLOptions = {}) {
-  const { returnPathname, screenHint, organizationId } = options;
+  const { returnPathname, screenHint, organizationId, redirectUri: redirectUriOptions } = options;
```

Comment:
nit: suggest singular `redirectUriOption` for the variable name. or... (maybe too fancy?):
```suggestion
  const { returnPathname, screenHint, organizationId, redirectUri = headers().get('x-redirect-uri') } = options;
```